client/src/App.js

```diff
@@ -31,6 +43,46 @@ function App() {
         </Col>
       </Row>
     </Container>
+  ); */
+
+  return (
+    <Router>
+      <Container>
+        <Row>
+          <NavbarComponent cart={1}/>   
+          {/*cart prop receives the length of the items list in shopping cart */}
+        </Row>
+        <Row className='vh-100 justify-content-center align-items-center'>
+          <Switch>
+            <Route exact path="/">
+              <ProducutListPage/>
+            </Route>
+            <Route exact path="/cart">
```

Comment:
I think it is needed because if I remove the 'exact' all the paths match with it (also /newclient for example). If I move this route at the end of the list without 'exact' and I go for example to a random route (for example /spg) it goes to the ProductListPage but in the URL still remains /spg.